Plan/common/src/main/resources/assets/plan/web/server.html

```diff
@@ -1539,7 +1539,12 @@ <h6>bStats Metrics</h6>
                 $('#calendar').text('');
                 onlineActivityCalendar('#calendar', json.data, json.firstDay);
                 $('#online-calendar-tab').click(function () {
-                    $('#calendar').fullCalendar('render');
+                    // Wrapping this in a 0ms setTimeout waits for all other event handlers
```

Comment:
Interesting how this is outside of the `onlineActivityCalendar `function call when comparing it to `sessionCalendar `function where the render is inside. I think the click event handler was originally for the same issue that this now handles - does it work if the `setTimeout `is moved inside the `onlineActivityCalendar `function similar to `sessionCalendar `- this way the click event might be unnecessary

Reply:
I just tested it and it seems like the click event is necessary. If it is not called correctly, the calendar looks like this when you load the page initially to a different tab and then switch:
![image](https://user-images.githubusercontent.com/31071265/96259742-080f7180-0f8c-11eb-9a0b-f7b1aa905a2e.png)
I figured out that this is likely because the JS that renders the calendar is called when the width is zero, before the tab is fully transitioned. That is why I added a `setTimeout` there, but it seems like we still need the click event.

Reply:
Alright let's keep it then